pkg/scaffold/v2/prometheus/monitor.go

```diff
@@ -1,7 +1,24 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package prometheus
 
 import (
 	"path/filepath"
+
```

Comment:
Minor nit: I completely agree with this blank line, I actually authored #1187, but I would not change it in this PR as it is not related to the same topic.

Reply:
Ok, will revert this.